### DISCUSSION TITLE
Auth problem

### DISCUSSION THREAD

**Discussion by damacus** [astrandb/miele#112] [Q&A]
👋🏼 firstly thanks for the integration 🥳 

When I add and configure the new Miele integration I get Authorization failed. 

Am I right that the account that you need to login with is the account on miele.co.uk (or whatever your local site is)?

**Comment by astrandb** [comment]
You can verify your login credentials at the Miele developer site https://www.miele.com/developer/swagger-ui/ -> Authorize

**Reply by damacus** [reply]
Yeah, indeed they were equally broken. This is a bit of a head-scratcher. Which creds am I supposed to be using?

**Reply by astrandb** [reply]
Are you using any special characters in your username/password? 
Others have reported <100% compatibility across platforms.

**Reply by damacus** [reply]
Ah yeah, there are some special chars in there. thanks for the pointers 🚀 

Actually if that is the case, the UK site requires special characters, so I'm not sure how that ever works, or is there a sub-set of social characters that don't work (I think you're saying the latter)

**Comment by squonk11** [comment]
It seems that I am having the same problem. I installed the Miele integration via HACS and now I am directed to a login page from Miele. But using my credentials I always get "Authentication failed". Using the same credentials I am able to login to the Miele [Miele](https://shop.miele.de/b2c_tp2/b2c/gotoPersonalDetails.do?xsrfid=xZCDH-0TJOhiM31HMOHt60xwbMkg1_en0PQ), then the login works perfectly. What am I doing wrong?
I am also able to login [here](https://www.miele.com/developer/swagger-ui/index.html) using my client_id and client_secret.

**Reply by rudizl** [reply]
Not sure, but are you using credentials from the google/apple application? They are not the same as the one for miele site.

**Reply by squonk11** [reply]
I am using the credentials I got when I integrated my new washing machine into the Miele cloud. Probably that are the ones for the google app. Where do I get the ones needed here?

**Reply by rudizl** [reply]
Maybe I am wrong, but registration into Miele cloud was true the application but not true the site.
